client/src/components/Upload.vue

```diff
@@ -41,7 +41,8 @@ export default {
     },
     /**
      * Takes the pending upload and returns the # of images or size of the file based on the type and state
-     * @param {Object} pendingUpload - contains , status, type
+     * @param {{ type: string, size: number, files: Array, uploading: boolean }} pendingUpload
+     * @returns {number} # of images or size of file depending on type and state
      *  size, and list of files to upload.
      */
```

Comment:
Line 46 should be removed.
line 43 is duplicate information and should be removed.

Reply:
sorry, not thinking properly thing morning.